keyboards/pauperboards/brick/info.json

```diff
@@ -0,0 +1,250 @@
+{
+    "keyboard_name": "Brick",
+    "manufacturer": "pauperboards",
+    "url": "https://p3dstore.com/products/cervello-pcb-only-group-buy",
+    "maintainer": "pauper",
+    "usb": {
+        "vid": "0xFEFE",
+        "pid": "0x0001",
+        "device_version": "0.2.0"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": [ "D3", "D5", "F0", "F1" ],
+        "cols": [ "B1", "B3", "F7", "F6", "D4", "D6", "D7", "B4", "B5", "B6", "C6", "C7" ]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bluetooth": false,
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "midi": false,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "unicode": false,
+        "lto": true
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "F4", "pin_b": "F5", "resolution": 1 }
```

Comment:
Some doubt here about a default resolution value of 1.

Reply:
Good catch. I fixed this.